docs/tool/config.js

```diff
@@ -3,15 +3,17 @@
  */
```

Comment:
This is meant to be a "config" file, I'd personally move all the logic to generate the manifest to `docs/tool/manifest.js` and leave here only the parts that are "configurable"

Reply:
I guess the line is not always easy to draw. We should probably rethink this config globally though. So don't take this comment as a big blocker.

Reply:
That is what I was going to do initially, but that file exports a single function that is very specific to the logic for the packages and data docs. I considered exporting an object from that file which contained more than one function but went the path of least resistance. I'd be happy to make some changes if you have a specific recommendation.

Reply:
Since the function is already doing two things, it can do three :) and we can split it into three functions if necessary.

I'd only keep `'packages/components/src'` as a config variable. Something like: `componentsSource` or something like that, and just move everything else to `manifest.js`.

Also it looks like the `baseRepoUrl` is already being used in that file (hardcoded), so it might be good to use this variable consistently.

Reply:
Let's address this feedback and 🚢 

I really like the automation proposed in this PR 💯